docs/code-reviews/rejected-suggestions/ef52094_migration-finish-core-cleanup_PR165_reserve-extra-byte.md

```
@@ -0,0 +1,24 @@
+Title: Reject reserving an extra byte for CBOR batch buffer
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)